README.md

```diff
@@ -2,12 +2,11 @@
   -- This file is auto-generated from README_js.md. Changes should be made there.
   -->
 
-
 # uuid [![CI](https://github.com/uuidjs/uuid/workflows/CI/badge.svg)](https://github.com/uuidjs/uuid/actions?query=workflow%3ACI) [![Browser](https://github.com/uuidjs/uuid/workflows/Browser/badge.svg)](https://github.com/uuidjs/uuid/actions?query=workflow%3ABrowser)
 
-For the creation of [RFC4122](https://www.ietf.org/rfc/rfc4122.txt) UUIDs
+For the creation of [RFC9562](https://www.rfc-editor.org/rfc/rfc9562.html) (formally [RFC4122](https://www.rfc-editor.org/rfc/rfc4122.html)) UUIDs
 
-- **Complete** - Support for RFC4122 version 1, 3, 4, 5, and 7 UUIDs
+- **Complete** - Support for RFC9562 version 1, 3, 4, 5, and 7 UUIDs
```

Comment:
Let's just commit to supporting all official RFC versions (1,3-8).  I'm working on v6 now, and v8 is trivial (just need to add support in `validate()` and `version()`)
```suggestion
- **Complete** - Support for all RFC9562  UUID versions
```

Reply:
Great idea. I'll take v8 while you're working on v6

Reply:
Sorry, 'didn't see this comment until just now.   I appreciate you creating the issue to kick off this work, though.  I should have done that.  My bad!